zkevm-circuits/src/util.rs

```diff
@@ -34,3 +36,28 @@ pub fn power_of_randomness_from_instance<F: Field, const N: usize>(
 
     power_of_randomness.unwrap()
 }
+
+pub(crate) fn build_tx_log_address(index: u64, field_tag: TxLogFieldTag, log_id: u64) -> Address {
+    (U256::from(index) + (U256::from(field_tag as u64) << 32) + (U256::from(log_id) << 48))
+        .to_address()
+}
+
+pub(crate) fn build_tx_log_from_expr_to_expression<F: halo2_proofs::arithmetic::FieldExt>(
```

Comment:
I think having a single function for the tx log expression is enough.  Could you rename this one to `build_tx_log_expression` and delete the second one?

Reply:
ok